### PR TITLE
Switch SDK to use WAMP

### PR DESCRIPTION
**Pull request by richardartoul**

### CODE REVIEW

File: hyperdash/constants.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+AUTH_HEADER_KEY = "x-hyperdash-auth"
+WAMP_ENDPOINT = "/api/v1/sdk/wamp"
+WAMP_REALM = u"hyperdash.v1.sdk"
+
+
+def get_base_url():
+    environment = os.environ.get("HYPERDASH_ENVIRONMENT")
+    if environment == "dev":
+        return u"ws://127.0.0.1:4000"
+    return u"wss://hyperdash.io"
```
Comment (spence) [inline hyperdash/constants.py]: Are we hard coding the URL here? Why not use an env for the URL (e.g., `HYPERDASH_SERVER`).

File: hyperdash/server_manager.py [hunk 2]
```diff
@@ -32,59 +71,54 @@ def received_message(self, m):
...
             except IndexError:
-                return
+                # Clean exit
+                returnValue(True)
 
             try:
-                self.send(message)
+                # TODO: Send multiple messages at once
+                yield self.call(
+                    u"sdk.sendMessage",
```
Comment (spence) [inline hyperdash/server_manager.py]: Why use unicode strings some places while not others?

File: test_script.py [hunk 3]
```diff
@@ -1,12 +1,12 @@
 import time
-import sys
 from hyperdash.sdk import monitor
 
 
-@monitor(job_name="ml_model")
+@monitor(job_name="cats v. dogs")
```
Comment (spence) [inline test_script.py]: Aren't we calling this `@hyperdash(...`?

File: hyperdash/constants.py (hunk 1 above)
Comment (richardartoul) [inline hyperdash/constants.py]: Good idea

File: test_script.py (hunk 3 above)
Comment (richardartoul) [inline test_script.py]: I think we're calling the function monitor and it will be in the hyperdash namespace

**Pull request merged**